cmd/launcher/main.go

```diff
@@ -51,7 +54,7 @@ func runLaunch() error {
 	}
 
 	if err := launcher.Launch(os.Args[0], os.Args[1:]); err != nil {
-		return cmd.FailErrCode(err, cmd.CodeLaunchError, "launch")
+		return cmd.FailErrCode(err, platform.CodeFor(cmd.LaunchError), "launch")
```

Comment:
Now that we have a `LifecycleExitError`, we could refactor this to prevent callers from passing an unsupported. Something to think about in the future.
```
return cmd.FailErrCode(err, platform, cmd.LaunchError, "launch")
```

Reply:
Right now it's very convenient that `cmd` doesn't depend on platform - because then we can have `cmd.LaunchError` without creating a circular dependency (cmd -> platform -> v05/v06 -> cmd). But if we decide to do this I'm sure we can find another place for those constants.